client/src/app/pipes/translate.pipe.ts

```
@@ -1,14 +1,40 @@
 
   transform(key: TranslationKey): string {
-    return key + this.locale;
+    const test = 213123123;
+
+    const maybe = (test * 10) as any;
+
+    // const test = this.isLocaleSupported() as any;
+    // console.log('test', test);
+    if (!this.isLocaleSupported()) return textMappings[key].en;
+    return (
+      textMappings[key][this.language() as TranslationLanguage] + '!' + maybe
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)